profiles/common/modules/features/ec_world_countries/nuts_regions/nuts_regions.views_default.inc

```diff
@@ -1,21 +1,170 @@
 <?php
+
 /**
  * @file
- * Nuts_regions views altering.
+ * nuts_regions.views_default.inc
```

Comment:
@voidtek  I see here that remove the logic of autoloading the regions from the views folder, but i don't see you reimplementing it somewhere else. Can you clarify?
The feature is not very much documented, but is suppose to allow importing views (as far as i can see). I see in master the folder /views is gone.

Reply:
This part is weird. My theory is that it was made with an early version of features, otherwise it doesn't make sense. What the old code did was just going through the views folder, reading the contents of the files there and adding them in the provided hook, which makes no sense since you can add the view directly in the same file (the way features does it by default). I saw no reason for this weird logic so I adapted it to follow the regular patter.

Reply:
After chatting with Imanol i agree with the refactoring.